appcircle_ios_reactnative-codepush/1.1.0/component.yaml

```diff
@@ -0,0 +1,77 @@
+platform: common
```

Comment:
The two files are exactly the same, so why did we separate them into Android and iOS?